src/Components/Pages/Legal/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,22 @@
+import { ContentfulPage } from "../../../contentful/ContentfulPage";
+import { Header } from "../../Header/Header";
+import privacyPolicy from "../../../data/privacy-policy.en.json";
+import "./Legal.scss";
+
+export const PrivacyPolicy: React.FC = () => {
+  return (
+    <div id="privacy-policy-page">
+      <Header
+        title="Privacy policy"
+        subtitle="Last modified: May 13, 2024"
```

Comment:
Since we added Good Cause NYC to the list of products

```suggestion
        subtitle="Last modified: Jan 24, 2025"
```